lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
I'm not deep into Rikki so I may be wrong here.
But I  think that Rikki should still be a class, since it is likely that there will be more than one of them?